tslint.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
Use eslint for the linter, not tslint.
Best would be to remove the lib and rerun the schematic with eslint.
You can run `nx generate @nrwl/workspace:remove --projectName=engine-theia` to remove all files changes